homeworks/vladimir.karbivnychiy_VladimirSkarb/homework_6/task1.js

```diff
@@ -0,0 +1,26 @@
+const user = {};
+Object.defineProperties(user, {
+    fullName: {
+        set: (name) => {
+            this.fullName = name.replace(/([[A-Za-z])\S/g, s => s.toLowerCase()).replace(/(^|\s|[-])\S/g, s => s.toUpperCase());
```

Comment:
arrow function will access global scope, `fullName` will be available globally, should be corrected